lms/djangoapps/learner_home/test_serializers.py

```diff
@@ -224,6 +225,18 @@ def test_missing_resume_url(self):
         # Then the resumeUrl is None, which is allowed
         self.assertIsNone(output_data["resumeUrl"])
 
+    def test_progress_url(self):
```

Comment:
This test won't always test the code we're trying to test, we should test both possibilities by mocking the `course_home_mfe_progress_tab_is_active` so that it returns true and false and make sure that the correct expected URL is provided in either case.

See [the quick guide](https://docs.python.org/3/library/unittest.mock.html#quick-guide) if you haven't played with mocking before, and I'm happy to help if you're not sure how to proceed.

Reply:
Sorry, my bad. Had totally forgotten about mocking. I'll provide the fix by tomorrow or the day after. Thanks for the review and mentioning @feanil.

Reply:
Quick question here. I can make this into two tests (one for when the function returns true and another for when it returns false) or just keep it as one test. Which one is preferred in your opinion @feanil?